core/src/main/kotlin/no/strooped/view/screen/JoinGameScreen.kt

```diff
@@ -9,12 +9,19 @@ import com.badlogic.gdx.scenes.scene2d.ui.Image
 import com.badlogic.gdx.utils.viewport.ScreenViewport
 import no.strooped.StroopedController
 import no.strooped.TextureSizes
+import no.strooped.model.GameRoom
+import no.strooped.view.screen.components.Label
 import no.strooped.view.screen.components.TextFieldInput
 import no.strooped.view.screen.components.UIButton
+
 /**
  * Uses https://otter.tech/an-mvc-guide-for-libgdx/ as inspiration
  * */
-class JoinGameScreen(private val controller: StroopedController) : Screen {
+class JoinGameScreen(
+    private val controller: StroopedController,
+    private val errorMessage: String? = null,
+    private val storedGame: GameRoom? = null
```

Comment:
Not sure if this we have time to use this, but the idea was to prefil the username and joinPin field, of GameRoom contained any data